### PR TITLE
Rolling Updates Doc covers config

### PR DESCRIPTION
**Pull request by nzoschke**
## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

File: _deployment/rolling-updates.md [hunk 1]
```diff
@@ -3,22 +3,85 @@ title: "Rolling Updates"
...
+If the Process does not [exposes ports](/docs/port-mapping) it is considered healthy if it starts and doesn't immediately exit or crash. 
+
+If the Process [exposes ports](/docs/port-mapping) is it considered healthy after it:
+
+* Registers behind a load balancer
+* Passes a network connection health check
+
+Common causes for not passing health checks are:
+
+* The cluster does not have sufficient memory or CPU resources available to start a new process
```
Comment (MiguelMoll) [inline _deployment/rolling-updates.md]: Might be worth mentioning a common issue I've seen when a process exposes a port but not enough instances to handle that.

**Comment by nzoschke** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;gppSSTdf&#x2F;605-document-deploy-process"> document deploy process</a>

File: _deployment/rolling-updates.md [hunk 2]
```diff
@@ -3,22 +3,84 @@ title: "Rolling Updates"
...
+    web:
+      labels:
+        - convox.health.path=/_health
+        - convox.health.port=5000
+        - convox.health.timeout=3
+      ports:
+        - 443:5000
+```
+
+The number of new processes to start can be configured for apps that need to start many processes within the 10m timeout:
```
Comment (ddollar) [inline _deployment/rolling-updates.md]: This is pretty light on details about why you might want to tune this and what you should set them to.

File: _deployment/rolling-updates.md [hunk 3]
```diff
@@ -3,22 +3,84 @@ title: "Rolling Updates"
...
+
+Common causes for not passing health checks are:
+
+* The cluster does not have sufficient memory or CPU resources available to start a new process
+* The cluster does not have sufficient instances where a new process port is not already reserved by an older release
+* A process crashes immediately after starting due to a problem in the latest code
+* A process takes too long to initialize its server and therefore fails a network health check
+
+#### Configuring Deployment Parameters
+
```
Comment (ddollar) [inline _deployment/rolling-updates.md]: This is pretty light on details about why you might want to tune this and what you should set them to.

File: _deployment/rolling-updates.md [hunk 4]
```diff
@@ -3,22 +3,84 @@ title: "Rolling Updates"
...
+If the Process does not [expose ports](/docs/port-mapping) it is considered healthy if it starts and doesn't immediately exit or crash.
+
+If the Process [exposes ports](/docs/port-mapping) is it considered healthy after it:
+
+* Registers behind a load balancer
+* Passes a network connection health check
+
+Common causes for not passing health checks are:
+
```
Comment (ddollar) [inline _deployment/rolling-updates.md]: How can I tell if any of these is happening to me?

**Pull request merged**